README.md

```diff
@@ -66,9 +66,13 @@ server.vm.synced_folder 'assets', '/assets', disabled: true
 
 Run 'vagrant up' from your machine (or 'vagrant provision' if your vbox is already running), then:
 
-On an OSX run:  `sudo mount -o resvport 192.168.50.5:/assets /assets`
+1. At the command line, cd into the vampd root directory and mkdir assets
```

Comment:
I am hesitant with the `assets` vs `/assets` because if you have gitsubmodules you will actually need the absolute path to be the same on your NFS mount of git commands don't work locally. It is a weird side effect. 

However the changes look good besides that.